src/pages/Dashboard/NoTeamRegistration.js

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from 'react';
+import InitialPage from '../../components/DashboardNoTeam/InitialPage/InitalPage';
+import CreateTeamForm from '../../components/DashboardNoTeam/CreateTeamForm/CreateTeamForm';
+import ConfirmTeamCreation from '../../components/DashboardNoTeam/ConfirmTeamCreation/ConfirmTeamCreation';
+
+const NoTeamRegistration = props => {
```

Comment:
Same comment related to destructuring props as `src/components/DashboardNoTeam/InitialPage/InitalPage.js`